src/runtime/schema.ts

```diff
@@ -125,6 +125,20 @@ export class Schema {
     return Schema._typeString(fieldType1) === Schema._typeString(fieldType2);
   }
 
+  static isAtLeastAsSpecificAs(fieldType1, fieldType2): boolean {
```

Comment:
What are the types of `fieldType1` and `fieldType2`? Are they just `Schema`?

Reply:
This method is called from the manifest parser, before the Schemas were process, so these are {kind:'schema-xyz', ...} objects